modules/porous_flow/src/auxkernels/PorousFlowPropertyAux.C

```diff
@@ -233,6 +252,14 @@ PorousFlowPropertyAux::computeValue()
     case PropertyEnum::PERMEABILITY:
       property = (*_permeability)[_qp](_k_row, _k_col);
       break;
+
+    case PropertyEnum::HYSTERESIS_ORDER:
+      property = 1.0 * (*_hys_order)[_qp];
```

Comment:
Why the 1.0 multiplier?

Reply:
Multiplying by 1.0 was my attempt to say to readers "hys_order" is an integer but we want a real.

Reply:
It is not necessary, property is defined as a Real above.
```suggestion
      property = (*_hys_order)[_qp];
```